### ISSUE TITLE
Switch licenses

### ISSUE THREAD

**Issue by ReperakDev** [i-like-trains72/optifine-resourcepack-conversion#11]
The GNU license used in this repository is for software, not documentation or text. I recommend you switch to [the CC Attribution 4.0 International license](https://creativecommons.org/licenses/by/4.0/), but you can use any media license you wish.

**Comment by i-like-trains72** [comment]
The reason I provide it under the GNU license I have used is because I provide _direct_ downloads of _mods_ and a resourcepack.
The resourcepack is ARR but I have permission to use it

**Comment by ReperakDev** [comment]
That's not how licenses work lmao

The repo itself it not software, and as such should use a works license.

**Comment by i-like-trains72** [comment]
The repo is literally providing software as a release.

**Comment by ReperakDev** [comment]
The repo itself is not software, you're redistributing it.

**Comment by i-like-trains72** [comment]
The only issue I see is that my repo is not software, but it provides software as a release. One of the pieces of software I include is ARR, and I am not allowed to do anything commercial with it to my knowledge. What would you suggest for that?

**Comment by i-like-trains72** [comment]
>  Share — copy and redistribute the material in any medium or format
Adapt — remix, transform, and build upon the material
for any purpose, even commercially

This is where I see an issue, specifically with the Adapt portion. That resourcepack is distributed in a release and is ARR. I had to get permissions to modify and redistribute it

**Comment by ReperakDev** [comment]
> The only issue I see is that my repo is not software, but it provides software as a release. One of the pieces of software I include is ARR, and I am not allowed to do anything commercial with it to my knowledge. What would you suggest for that?

You really shouldn't be redistributing these mods and resource packs at all, as it takes away from the creator(s). I suggest you link to the download page. It doesn't take a rocket scientist to install mods individually.

**Comment by i-like-trains72** [comment]
You're right, I could host just MCPP here as that mod needs to be built from source and I am not allowed to host it on Curseforge.
However, FSB is no longer being updated on Curseforge. 
Plus, it's much easier for newbies to do this instead of going back and forth between modrinth, CF, and github.
If Colormatic moves to modrinth I'll probably do just that. 

Right now, I can provide links to the CF page for the mods and the resourcepacks.

**Comment by ReperakDev** [comment]
> Plus, it's much easier for newbies to do this instead of going back and forth between modrinth, CF, and github.

The consideration is a kind gesture, but you underestimate the intelligence of the average user. In order to even get Fabric running a user must
1. Install a standalone Java runtime to launch the Fabric installer (unless they use MultiMC)
2. Change the profile in the Minecraft launcher
3. Navigate to the MC game folder
4. Install Sodium (I infer that this guide is targeted towards existing Sodium users)
If a user is able to follow instructions from a tutorial, they're able to follow your instructions.

tl;dr if a user is able to get Fabric even running, they're probably smart enough to be able to obtain resources on their own

**Comment by i-like-trains72** [comment]
good point.

**Comment by i-like-trains72** [comment]
stale issue
also many people use GDL and if people use the vanilla launcher well...they will prolly be able to do it.
*support for TLauncher users not provided*

**Comment by ReperakDev** [comment]
> stale issue

Stale issue? That's not how GitHub works. Issues are meant to stay up until the problem is *resolved*.

**Comment by i-like-trains72** [comment]
cc-by-sa good enough?
